luigi/static/visualiser/js/graph.js

```diff
@@ -242,6 +242,15 @@ Graph = (function() {
         return element;
     }
 
+    function escapeHtml(unsafe) {
+        return unsafe
```

Comment:
can `unsafe` be null? 
```suggestion
        if (unsafe == null) {
          return unsafe;
        }
        return unsafe
```

Reply:
Not really, it should better be a bug on the caller side, instead of hiding it I believe.